nuagekubemon/policy/resourcemanager.go

```diff
@@ -326,10 +326,20 @@ func (rm *ResourceManager) translatePeerPolicy(peer networkingV1.NetworkPolicyPe
 	if peer.NamespaceSelector != nil && peer.PodSelector != nil {
 		return fmt.Errorf("Unsupported network policy. Both pod and ns selectors specified")
 	}
-	if err := rm.findNamespacesWithLabel(peer.NamespaceSelector, namespaceLabelsMap); err != nil {
-		glog.Errorf("finding namespaces from selector label %s failed: %v", peer.NamespaceSelector.String(), err)
-		return err
+
+	if peer.NamespaceSelector != nil {
```

Comment:
can we move this logic inside `findNamespacesWithLabel` function? similar to `createPgAddVports`